src/sbvr-api/permissions.ts

```diff
@@ -387,12 +391,52 @@ const $checkPermissions = (
 	});
 };
 
-const constrainedPermissionError = new PermissionError();
-const generateConstrainedAbstractSql = (
+const convertToLamda = (filter: AnyObject, identifier: string) => {
+	const replaceOject = (object: AnyObject) => {
+		if (_.isString(object)) {
```

Comment:
I think
```suggestion
		if (typeof object === 'string') {
```
is preferable nowadays (typescript validates it from memory)